### ISSUE TITLE
Doesn't see `relude-0.3.0`

### ISSUE THREAD

**Issue by chshersh** [luite/hdiff#2]
First, I would like to tell, that I really like the idea of this website!

But I want to submit package synchronization issue.

Currently I see that the latest version of `relude` package on `hdiff` is `0.2.0`:

* http://hdiff.luite.com/cgit/relude/

When in reality the latest version is `0.3.0`:

* http://hackage.haskell.org/package/relude

It was uploaded to Hackage at `Sun Sep 9 12:18:25 UTC 2018`.

I see more recent packages already got synchronized, like `neat-interpolation`. The latest version is `0.3.2.4` and it was uploaded at `Fri Sep 14 01:47:56 UTC 2018`.

* http://hdiff.luite.com/cgit/neat-interpolation/

**Comment by chshersh** [comment]
@luite Sorry for bothering you again. Any updates on the issue? I still can't see new versions of the `relude` on `hdiff` :disappointed: I think that the idea of `hdiff` is very cool! And I'm looking forward to using it :slightly_smiling_face:

**Comment by juhp** [comment]
Yeah hdiff is _really_ useful.  As a distro packager I find it particularly helpful.

I noticed that all stack-2.x are missing too - and I have seen it occasionally for the odd package version.
I guess it skips a beat once in a while? Or is there any version cascading effect?

**Comment by LeventErkok** [comment]
Same thing for sbv. On hackage, right now it has version 8.9; on hdiff, latest is 8.7; which is about 5 months old by now.
